src/pages/clients-list/forms/ClientTemplateForm.js

```diff
@@ -605,6 +587,37 @@ const ClientTemplateForm = ({ setErrorMessage, recordId, _labels, plantId, maxAc
       setProfessionFilterStore(filteredList)
     }
   }
+  const { userId } = JSON.parse(window.sessionStorage.getItem('userData'))
+
+  const onClose = async () => {
+    const values = clientIndividualFormik.values
+    try {
+      const { record: cashAccountRecord } = await getRequest({
```

Comment:
why do we need cashAccountId ? we are not saving it, so for SURE we don't need it when closing the record.